include/zephyr/bluetooth/buf.h

```diff
@@ -85,13 +85,42 @@ struct bt_buf_data {
 #define BT_BUF_ISO_RX_COUNT 0
 #endif /* CONFIG_BT_ISO */
 
+/* see Core Spec v6.0 vol.4 part E 7.4.5 */
+#define BT_BUF_ACL_RX_COUNT_MAX 65535
+#define BT_BUF_ACL_RX_COUNT_DEFAULT 6
```

Comment:
`BT_BUF_ACL_RX_COUNT_DEFAULT` unused?

Reply:
My bad, I used it in the beginning but forgot to remove it, thank you!